app/views/users/_form.html.erb

```diff
@@ -1,29 +1,29 @@
 <%= form_for(@user) do |f| %>
   <% if params[:action] != 'edit' %>
     <div class="form-group col-xs-12">
-      <label class="col-xs-2 control-label" for="first_name">First Name</label>
+      <label class="col-xs-2 control-label" for="first_name">First Name*</label>
       <div class="col-xs-6">
         <%= text_field :user, :first_name, size: 20, placeholder: "Jane", class:"form-control", required: true %>
       </div>
     </div>
 
     <div class="form-group col-xs-12">
-      <label class="col-xs-2 control-label" for="last_name">Last Name</label>
+      <label class="col-xs-2 control-label" for="last_name">Last Name*</label>
       <div class="col-xs-6">
         <%= text_field :user, :last_name, size: 20, placeholder: "Doe", class:"form-control", required: true %>
       </div>
     </div>
 
     <div class="form-group col-xs-12">
-      <label class="col-xs-2 control-label" for="spire_id">Spire ID</label>
+      <label class="col-xs-2 control-label" for="spire_id">Spire ID*</label>
       <div class="col-xs-6">
         <%= text_field :user, :spire_id, size: 20, placeholder: "1234567", class:"form-control", required: true %>
       </div>
     </div>
   <% end %>
 
   <div class="form-group col-xs-12">
-    <label class="col-xs-2 control-label" for="department_id">Department</label>
+    <label class="col-xs-2 control-label" for="department_id">Department*</label>
     <div class="input-group col-xs-6">
       <%= f.select :department_id, options_from_collection_for_select(@departments, "id", "name", @user.department_id), {}, class: "selectpicker form-control", onchange: "update_disclaimer(this)", required: true %>
```

Comment:
Can we get line length under control a little here? I'm not going to be a stickler for exactly 80 characters but this is insane.